spec/delayed/serialization/active_record_spec.rb

```diff
@@ -3,13 +3,13 @@
 describe ActiveRecord do
   it "loads classes with non-default primary key" do
     expect {
-      YAML.load(Story.create.to_yaml)
+      YAML.load_dj(Story.create.to_yaml)
     }.not_to raise_error
   end
 
   it "loads classes even if not in default scope" do
     expect {
-      YAML.load(Story.create(scoped: false).to_yaml)
+      YAML.load_dj(Story.create(scoped: false).to_yaml)
```

Comment:
This spec just exists to make sure that ActiveRecords work with the psych ext. The new safe YAML.load in Ruby 3.1 and 3.2 requires extra configuration if you want to (de)serialize custom classes, but instead we can use the patched method that this gem actually uses for the same effect as before.